fixtures/emptyFile.JSON

```diff
@@ -0,0 +1,21 @@
+[
```

Comment:
Do you mean for the name of the file? The file is empty because the title and text key have no value(s).